<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XViewerClass</class>
 <widget class="QWidget" name="XViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xviewer.qrc">
    <normaloff>:/XViewer/img/xv.ico</normaloff>:/XViewer/img/xv.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 按钮样式 */
QPushButton:hover
 { 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 102, 184, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(0, 102, 184, 255));
border: none;
border-radius:8px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 50, 150, 255), stop:0.495 rgba(0, 102, 184, 255), stop:0.505 rgba(0, 102, 184, 255), stop:1 rgba(0, 50, 150, 255));
 border: none;
border-radius:8px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }




/* 顶部样式 */
#head
{
	/*background-color: rgb(255, 170, 0);*/
	background-color: rgb(35, 35, 35);
}

/* logo样式 */
#logo
{
	background-image: url(:/XViewer/img/logo_150_40.png);
}

/* 关闭按钮样式 */
#close
{
	background-image: url(:/XViewer/img/close.png);
}

/* 最小化按钮样式 */
#min
{
	background-image: url(:/XViewer/img/min.png);
}

/* 最大化样式 */
#max
{	
	background-image: url(:/XViewer/img/max.png);
}

/* normal样式 */
#normal
{	
	background-image: url(:/XViewer/img/normal.png);
}

/* body样式 */
#body
{
	
	/*background-color: rgb(35, 35, 35);*/
	background-color: rgb(111, 111, 111);
}

/* left样式 */
#left
{
	
	background-color: rgb(81, 81, 81);
}

/* cams样式 */
#cams
{
	
	background-color: rgb(56, 56, 56);
	/*background-color: rgb(81, 81, 81);*/
}

#status
{
	color: rgb(255, 0, 0);
	
	font: 12pt &quot;黑体&quot;;
}

#preview
{
	background-color: rgb(35, 35, 35);
}

#preview:checked
{
	background-color: rgb(0, 102, 184);
}

#playback
{
	background-color: rgb(35, 35, 35);
}

#playback:checked
{
	background-color: rgb(0, 102, 184);
}</string>
  </property>
  <widget class="QWidget" name="head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="head_button" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="max">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="min">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>close</zorder>
    <zorder>min</zorder>
    <zorder>normal</zorder>
    <zorder>max</zorder>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>监控中...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preview">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playback">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>回放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="body" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>791</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QWidget" name="left" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QListWidget" name="cam_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>200</width>
       <height>551</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="add_cam">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_cam">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>5</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_cam_3">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>5</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cams" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>461</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="playback_widget" native="true">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>0</y>
      <width>611</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="XCalendar" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>431</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="timelist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>161</width>
       <height>531</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>test</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XCalendar</class>
   <extends>QCalendarWidget</extends>
   <header>xcalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>MaxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normal</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>NormalWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>903</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>AddCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>-100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>-116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam_3</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>DelCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>-93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preview</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>-89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playback</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Playback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>-101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cam_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectCamera(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>-171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>-231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timelist</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>PlayVideo(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>-216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MaxWindow()</slot>
  <slot>NormalWindow()</slot>
  <slot>AddCam()</slot>
  <slot>SetCam()</slot>
  <slot>DelCam()</slot>
  <slot>Preview()</slot>
  <slot>Playback()</slot>
  <slot>SelectCamera(QModelIndex)</slot>
  <slot>SelectDate(QDate)</slot>
  <slot>PlayVideo(QModelIndex)</slot>
 </slots>
</ui>
